src/qt/editaddressdialog.cpp

```diff
@@ -11,6 +11,8 @@
 #include <QDataWidgetMapper>
 #include <QMessageBox>
 
+extern OutputType g_address_type;
```

Comment:
Can also be done in a follow up PR, but directly accessing the underlaying non QT system should be avoided. This should be accessed via WalletModel.